.eslintrc.js

```diff
@@ -64,4 +65,15 @@ module.exports = {
     quotes: [ 'error', 'single', { allowTemplateLiterals: true } ],
     semi: [ 'error', 'always' ]
   },
+  overrides:[
+    {
+      files:['docs/snippets/**/*.ts'],
+      rules:{
+        '@typescript-eslint/no-explicit-any': 'warn',
+        '@typescript-eslint/no-unused-vars': 'warn',
+        '@typescript-eslint/explicit-member-accessibility': 'warn',
+        '@typescript-eslint/no-var-requires': 'warn'
```

Comment:
Hi @saragerion . You mean as scope the Summary of #1252 , right ?

> Once all the code has been extracted, and https://github.com/awslabs/aws-lambda-powertools-typescript/pull/1251 has been merged, we should apply the same linting that the rest of the project has to these files.

Is the question why i have applied overrides in the previous [commit ](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/1259/commits/07c548d1cdf217bcfa734f1a9c12ef7a9f693931) of this PR ?
 
Because in the last [commit ](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/1259/commits/cc8d108424c397ee87c7f7cdf53da01032ff668a) i have  removed the overrides . 

Reason why i applied overrides section is described in the `description of changes` of this [PR](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/1259#issue-1551858001). 

Motivation:

> introduces overrides section in eslint configuration file in root of repository, so that we can discuss explicit about typescript linting issues for code snippets: no-explicit-any, no-unused-vars, explicit-member-accessibility, no-var-requires

I think that gave a better understanding on the differences between code and code-snippet for docs. 
If you read #1252  in Summary its is described the following:

> Notes
in many cases the signature of the handler function contains the any type. I would recommend to type them as unknown for the time being.

However with the PR we identified that it is not only the `any type`. It is also  the 
- no-explicit-any, no-unused-vars, explicit-member-accessibility, no-var-requires

that should be handled in code snippets. 

So i provided overrides section to make explicit the 2 options we can have 
1. apply same `eslint rules` and a subset of `rules` as overrides for the code snippets
2. change the code in the code snippets , so that the existing `eslint rules`  apply to code snippets.

We chosen `option 2` and changed the code in  the code snippets. 

Note:
`Option 1` has been inspired by the article of the [eslint author himself ](https://eslint.org/blog/2022/08/new-config-system-part-1/), for the reasons described [here ](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/1259#discussion_r1085763891)